internal/runnerinstall/ecs.go

```diff
@@ -1,6 +1,3 @@
-// Copyright (c) HashiCorp, Inc.
-// SPDX-License-Identifier: MPL-2.0
-
```

Comment:
I think we need these

Reply:
Not sure how I managed to remove those - added back.